tools/upgrade.sh

```diff
@@ -17,7 +17,7 @@ else
   RESET=""
 fi
 
-cd "$ZSH"
+builtin cd "$ZSH"
```

Comment:
`upgrade.sh` is ran with `sh`, `builtin` doesn't exist here.

Reply:
ok, i will remove this one.

Reply:
done